guardrails/schema/string_schema.py

```diff
@@ -122,8 +122,9 @@ def get_reask_setup(
 
         return self, prompt, instructions
 
-    def parse(self, output: str, **kwargs) -> Tuple[Any, Optional[Exception]]:
-        return output, None
+    def parse(self, output: str, **kwargs) -> Tuple[Any, bool]:
+        error = not output  # True if empty/None, else False
```

Comment:
Rather than true can we make this a ValueError with a message saying why we are considering parsing to have failed?  Like `output was empty` or some better equivalent.